certified-connectors/DocuSignDemo/script.csx

```diff
@@ -737,18 +737,33 @@ public class Script : ScriptBase
 
             string[] tabTypes = { "textTabs", "fullNameTabs", "dateSignedTabs", "companyTabs", "titleTabs", "numberTabs",
               "ssnTabs", "dateTabs", "zipTabs", "emailTabs", "noteTabs", "listTabs", "firstNameTabs", "lastNameTabs", "emailAddressTabs",
-              "formulaTabs" };
+              "formulaTabs", "checkboxTabs", "radioGroupTabs" };
             foreach (var tabType in tabTypes)
             {
               var tabStatusArray = tabs[tabType];
-
               foreach (var tab in tabStatusArray as JArray ?? new JArray())
               {
                 if (tab is JObject)
                 {
-                  var tabLabel = (string)tab["tabLabel"];
+                  if (tabType.Equals("checkboxTabs"))
+                  {
+                    if (newTabs[(string)tab["tabLabel"]] == null)
```

Comment:
Can the user set a custom tab label and have you verified it shows up in the trigger output ?

Reply:
Yes, User given labels will be returned as output with this implementation.